conversions.js

```diff
@@ -32,26 +30,10 @@ var convert = module.exports = {
 
 // hide .channels and .labels properties
 for (var model in convert) {
-	if (convert.hasOwnProperty(model)) {
```

Comment:
removed those checks, because it's always using plain objects, so prototype issues would not happen